components/apimgt/org.wso2.carbon.apimgt.rest.api.admin/src/main/resources/admin-api.yaml

```diff
@@ -2383,6 +2387,57 @@ paths:
           schema:
             $ref: '#/definitions/MonetizationUsagePublishInfo'
 
+######################################################
+  # The "Workflow approval" resource API
+  ######################################################
+  /workflows/update-workflow-status:
+
+    #-------------------------------------------------------------------
+    # Resume the workflow by approving or rejecting the workflow request
+    #-------------------------------------------------------------------
+    post:
+      x-scope: apim:api_workflow
+      x-wso2-curl: "curl -k -H \"Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8\" -H \"Content-Type: application/json\" -X POST -d @data.json \"https://localhost:9443/api/am/publisher/v0.14/workflows/update-workflow-status?workflowReferenceId=56e3a170-a7a7-45f8-b051-7e43a58a67e1\""
+      x-wso2-request: "POST https://localhost:9443/api/am/publisher/v0.14/workflows/update-workflow-status?workflowReferenceId=56e3a170-a7a7-45f8-b051-7e43a58a67e1\nAuthorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8\nContent-Type: application/json\n\n{\n   \"status\" : \"APPROVED\",\n   \"attributes\" : {\n      \"apiCurrentState\": \"Created\",\n      \"apiLCAction\": \"Publish\",\n      \"apiName\":\"APIname\",\n      \"apiVersion\" : \"1.0.0\",\n      \"apiProvider\" : \"admin\",\n      \"invoker\": \"admin\"\n   }\n}"
+      x-wso2-response: "HTTP/1.1 200 OK"
+      summary: Update workflow status
+      description: |
+        This operation can be used to approve or reject a workflow task.
+      parameters:
+        - $ref: '#/parameters/workflowReferenceId-Q'
+        - in: body
+          name: body
+          description: |
+            Workflow event that need to be updated
+          required: true
+          schema:
+            $ref: '#/definitions/Workflow'
+      tags:
+        - Workflows (Individual)
+      responses:
+        200:
+          description: |
+            OK.
+            Workflow request information is returned.
+          schema:
+            $ref: '#/definitions/Workflow'
+          headers:
+            Content-Type:
+              description: |
+                The content type of the body.
+              type: string
+        400:
+          description: |
+            Bad Request.
+            Invalid request or validation error.
+          schema:
+            $ref: '#/definitions/Error'
+        404:
+          description: |
+            Not Found.
+            Workflow for the given reference in not found.
```

Comment:
this sentence needs to be corrected

Reply:
fixed in https://github.com/wso2/carbon-apimgt/pull/7064/commits/977477bd23bb1ea19308a0e3f468b4157beac1c7